src/ansys/additive/core/parametric_study/parametric_runner.py

```diff
@@ -51,12 +51,16 @@ def simulate(
         additive: Additive,
         type: list[SimulationType] = None,
         priority: int = None,
+        iteration: int = None
     ) -> list[SingleBeadSummary, PorositySummary, MicrostructureSummary]:
         """Run the simulations in the parametric study with ``Status`` equal to ``Pending``.
 
         Execution order is determined by the ``Priority`` value assigned to the simulations.
         Lower values are interpreted as having higher priority and are run first.
 
+        If the ''Iteration'' value is specified, only simulations with the specified iteration 
+        number are run.
+
```

Comment:
Let's get rid of this. This is general information about the method. Priority is mentioned because it impacts the operation of the method even when the parameter is not provided.